app/forms/idv/api_image_upload_form.rb

```diff
@@ -154,6 +154,7 @@ def extra_attributes
 
       @extra_attributes[:front_image_fingerprint] = front_image_fingerprint
       @extra_attributes[:back_image_fingerprint] = back_image_fingerprint
+      @extra_attributes[:liveness_checking_required] = liveness_checking_required
```

Comment:
It feels weird to include this here as a way to get the `liveness_checking_required` value in the `image_upload_response_presenter`. Is there a better way?

Reply:
@charleyf or you can just grab it from response in the presenter. Not sure about the impact on logging perspective.

Reply:
Is there an easy way to do that? This sounds like what I want, but I'm not sure how to implement it.

Reply:
Came up that adding this here actually addresses some other needs we have around logging: https://cm-jira.usa.gov/browse/LG-12160

Reply:
@charleyf, need some changes which I think will change _presenter's_ design purpose.  In my view, the _presenter_ is kind of response/data transformer which should only function with the data provided(_@form_response_).

Reply:
> the presenter is kind of response/data transformer which should only function with the data provided(@form_response)

This makes sense to me, can you tell me more about what changes you're suggesting?

Reply:
@charleyf, not suggesting further change here, what I meant if we do changes(by passing additional data other then @form_response to the presenter, likely in the initializer) that will not conform to the designed purpose of the presenter.

Reply:
Ah, ok. I think you're saying:
- This is good the way it is for this PR.
- But, for future work, it's good to keep in mind that we should not be passing new things to the presenter to keep that class clean.